plenum/test/helper.py

```diff
@@ -1222,7 +1222,8 @@ def create_pre_prepare_params(state_root,
                               pp_seq_no=0,
                               inst_id=0,
                               audit_txn_root=None,
-                              reqs=None):
+                              reqs=None,
+                              bls_multi_sigs=[]):
```

Comment:
I think we need to include `bls_multi_sigs` into all test helpers for creation of PrePrepare. If it's not specified, a defualt can be added (like `[bls_sig]`).